src/array.vala

```diff
@@ -2,6 +2,9 @@ using GLib;
 
 public class Vast.Array : Object
 {
+    public const ssize_t NEW_AXIS = ssize_t.MAX;
+    public const size_t DEFAULT_DIMENSION = size_t.MAX;
```

Comment:
Just do `arr.redim (5).transpose ({0, 1, 4, 2, 3})`. Basically add new axis and transpose them to the middle.